src/Metadata/Driver/YamlDriver.php

```diff
@@ -32,27 +36,74 @@ class YamlDriver extends AbstractFileDriver
      * @var PropertyNamingStrategyInterface
      */
     private $namingStrategy;
+    /**
+     * @var FileLocatorInterface|FileLocator
```

Comment:
Do we need the `FileLocator` typehint here?

Reply:
Not really, just copy-pasted from `metadata`. Will fix it